test/stablex/batch_exchange.js

```diff
@@ -2044,14 +2044,16 @@ contract("BatchExchange", async accounts => {
   })
   describe("Large Examples", () => {
     it("ensures hard gas limit on largest possible ring trade ", async () => {
-      const batchExchange = await setupGenericStableX(25)
+      const batchExchange = await setupGenericStableX(30)
       const sixPointFiveMillion = 6500000
-      await makeDeposits(batchExchange, accounts, longRingTrade.deposits)
+
+      const tradeExample = largeRing30
+      await makeDeposits(batchExchange, accounts, tradeExample.deposits)
       const batchId = (await batchExchange.getCurrentBatchId.call()).toNumber()
-      const orderIds = await placeOrders(batchExchange, accounts, longRingTrade.orders, batchId + 1)
+      const orderIds = await placeOrders(batchExchange, accounts, tradeExample.orders, batchId + 1)
```

Comment:
this should be using largeRing30, shouldn't it?

Reply:
He is setting 
const tradeExample = largeRing30, 
so I think it is good.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FK0Hy2NwI8IXZK%2Fgiphy.gif&token=blind" data-gitmeme-token="blind"><img src="https://media2.giphy.com/media/K0Hy2NwI8IXZK/giphy.gif" title="Created by gitme.me with /blind"/></a>

I see now. But WHY?

We also need to make sure that the second solution actually attributes deposits to the balances that are traded in it. So either the users or the tokens need to be different than in the first solution.

Reply:
We should also make sure that all users have a pending withdraw requests on all tokens that gets "blocked" when their balance increases to maximize the worst case scenario.

Reply:
oh my... I will have to think about this a bit.